internal/host/hostutil/host_utils.go

```diff
@@ -40,6 +40,28 @@ func GetHostnameForMsg(host *models.Host) string {
 	return hostName
 }
 
+func IsBeforeInstallation(status string) bool {
+	beforeInstallation := []string{models.HostStatusDiscovering, models.HostStatusKnown,
+		models.HostStatusDisconnected, models.HostStatusInsufficient,
+		models.HostStatusPendingForInput}
+	return funk.ContainsString(beforeInstallation, status)
+}
+
+func IsInfraEnvStatus(status string) bool {
+	infraEnv := []string{models.HostStatusDisconnectedUnbound,
+		models.HostStatusInsufficientUnbound, models.HostStatusDiscoveringUnbound,
+		models.HostStatusKnownUnbound}
+	return funk.ContainsString(infraEnv, status)
+}
+
+func IsBoundStatus(status string) bool {
```

Comment:
i think that you can simply add HostStatusBinding to IsBeforeInstallation function and make IsBoundStatus redundant